docs/dictionary/function/byteOffset.lcdoc

```diff
@@ -15,17 +15,55 @@ OS: mac, windows, linux, ios, android
 Platforms: desktop, server, mobile
 
 Example:
-offset("c","abcde") -- returns 3
+byteOffset("c","abcde") -- returns 3
 
 Example:
-offset("b","abacadabra",2) -- returns 6
+byteOffset("b","abacadabra",2) -- returns 6
 
 Example:
-offset("bark","embarking") -- returns 3
+byteOffset("bark","embarking") -- returns 3
 
-References: find (command), sort container (command),
-wordOffset (function), itemOffset (function), length (function),
-lineOffset (function), offset (function), begins with (operator)
+Example:
+byteOffset(null & numberToByte(255),"End" & null & numberToByte(255)) -- returns 4
+
+Parameters:
+bytesToFind (string): a string of binary values to find
+
+
+binaryStringToSearch (string): a container with binary values
+
+
+bytesToSkip:
+A non-negative integer. If you don't specify how many bytesToSkip, the
+byteOffset function does not skip any bytes and starts at the beginning of
+the binaryStringToSearch.
 
-Tags: text processing
+Returns:
+The <byteOffset> <function> <return|returns> a <non-negative> <integer>.
+
+Description:
+Use the <byteOffset> <function> to find where a <binaryString> occurs in another
+<binaryString>. 
```

Comment:
`<binaryString>` won't work as a reference because it isn't a parameter name or dictionary entry. You could potentially change this to 
`...find where <bytesToFind> occurs in <binaryStringToSearch>`, or
`...find where a string of <binary data> occurs in another string of <binary data>`